tmail-backend/jmap/extensions/pom.xml

```diff
@@ -24,6 +24,16 @@
             <groupId>${project.groupId}</groupId>
             <artifactId>team-mailboxes</artifactId>
         </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
```

Comment:
Rather than creating all the code in `tmail-backend/jmap` wouldn't it be better to create a module named `tmail-backend/tmail-third-party/openpaas` ?